packages/models-library/src/models_library/projects_state.py

```diff
@@ -99,6 +91,23 @@ def check_status_compatible(cls, v, values):
             raise ValueError(msg)
         return v
 
+    @validator("owner", always=True)
```

Comment:
this should be a root_validator, as you are validating the whole module. this would also be safer since you would not be playing with strings but with the fields directly.

Reply:
modified to root validator